ipinfo/cmd_asn.go

```diff
@@ -6,10 +6,31 @@ import (
 	"net/http"
 
 	"github.com/fatih/color"
+	"github.com/ipinfo/cli/lib/complete"
+	"github.com/ipinfo/cli/lib/complete/predict"
 	"github.com/ipinfo/go/v2/ipinfo"
 	"github.com/spf13/pflag"
 )
 
+var completionsASN = &complete.Command{
+	Flags: map[string]complete.Predictor{
+		"-t":        predict.Nothing,
+		"--token":   predict.Nothing,
+		"--nocache": predict.Nothing,
+		"-h":        predict.Nothing,
+		"--help":    predict.Nothing,
+		"-f":        predict.Set(coreFields),
+		"--field":   predict.Set(coreFields),
```

Comment:
Bad copy paste! Wrong field set

Reply:
My bad! I forgot about the ASN fields. It is fixed now.